src/frontend/src/btc/services/btc-listener.services.ts

```diff
@@ -19,13 +19,15 @@ export const syncWallet = ({
 		}
 	} = data;
 
-	balancesStore.set({
-		tokenId,
-		data: {
-			data: BigNumber.from(balance),
-			certified
-		}
-	});
+	if (nonNullish(balance)) {
```

Comment:
If the balance is nullish, shouldn't we reset the store otherwise we keep in memory the previous balance?

Reply:
Good point, updated.